src/archivematica/storage_service/storage_service/settings/base.py

```diff
@@ -652,6 +652,11 @@ def _get_settings_from_file(path):
 
     OIDC_RP_SIGN_ALGO = environ.get("OIDC_RP_SIGN_ALGO", "HS256")
 
+    OIDC_USE_PKCE = environ.get("OIDC_USE_PKCE", "false")
```

Comment:
```suggestion
    OIDC_USE_PKCE = is_true(environ.get("OIDC_USE_PKCE", "false"))
```

Reply:
@replaceafill: I struggled with this with my own bool setting on the other PR - seems you can't specify an unquoted bool value in a docker compose file? I could not get it to work. The technique above is what is used in other areas of AM. See the following code to Activate OIDC in AM and SS:

https://github.com/artefactual/archivematica-storage-service/blob/qa/0.x/tests/integration/docker-compose.yml#L35
https://github.com/artefactual/archivematica-storage-service/blob/qa/0.x/src/archivematica/storage_service/storage_service/settings/base.py#L575

I don't see any reason to change this.

Reply:
> But I think when the SS settings module reads the environment it's all strings

From https://docs.python.org/3.9/library/os.html#os.environ:

> os.environ
> A [mapping](https://docs.python.org/3.9/glossary.html#term-mapping) object where keys and values are strings that represent the process environment.

Reply:
I realise now I misread your comment @replaceafill! :man_facepalming:  Yes this instance should use is_true(). I will add a commit...